contexts/ArrayWrapper.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC, ReactNode, useState, createContext, useEffect } from 'react'
+
+type FilterArray = [Array<any>, boolean]
+
+export const FilterArrayContext = createContext<FilterArray>([[], false])
+
+enum State {
+    ShowAll = 'Show All',
+    ShowLess = 'Show Less',
+}
+
+export const ArrayWrapper: FC<{ array: Array<any>; children: ReactNode; limit: number }> = ({
```

Comment:
```suggestion
export const ArrayWrapper: FC<{ array: Array<any>; children: ReactNode; minimum: number }> = ({
```
`limit` --> `minimum` I feel is more clear